docs/tutorial/introduction.md

```diff
@@ -2,82 +2,119 @@
-- We demonstrate here how to program the accelerator of interest.
+
+- We demonstrate how to program the accelerator of interest.
 - We also show how to build the program and run simulations.
 
 7 - [Other Tools](./other_tools.md)
+
 - There exists tools that can help profile and analyze your design.
 
 8 - [Other Designs](./more_designs.md)
-- We've used our SNAX platform for various accelerators already - In this section, we demonstrate these accelerators and how we profiled them.
+
+- We've used our SNAX platform for various accelerators already.
+- In this section, we demonstrate these accelerators and how we profiled them.
 - We also show synthesized results of our work to see how the designs turn out.
 
 # Getting Started
 
-To follow along with the tutorial, we recommend to create a GitHub Codespace. Alternatively, you can clone the repository locally and use our prebuilt docker container or install the requirements locally yourself.
+To follow along with the tutorial, we recommend to **create a GitHub Codespace**. Alternatively, you can **clone the repository locally** and use our prebuilt docker container.
 
 ## Github Codespace
 
+!!! note
+
+    It is recommended you open Codespace in Chrome.
+
 Opening a Github Codespace is the most convenient way to get started quickly. To create a new codespace, go to the main repository, and follow code -> codespace -> create codespace. Or access this at [SNAX cluster codespace](https://codespaces.new/KULeuven-MICAS/snitch_cluster).
 
-![image](https://github.com/KULeuven-MICAS/snitch_cluster/assets/47864363/27f3c084-ba26-4653-ad68-d9e898ca0597)
+![image](https://github.com/KULeuven-MICAS/snax_cluster/assets/26665295/2831e951-84ec-4909-a3a1-e3eef816c56e)
+
+This will launch a new window where the container image for the codespace will be built. This can take a couple of minutes but you only run this once. After this, a Visual Studio Code client will be launched in your browser. This system has all the requirements for developing, building, and simulating a SNAX Cluster preinstalled. If you prefer this, you can launch this codespace in the Visual Studio Code desktop client.
 
-This will launch a new window where the container image for the codespace will be built. This can take a couple of minutes, but is only required to run once. After this, a Visual Studio Code client will be launched in your browser. This system has all the requirements for developing, building and simulating a SNAX Cluster preinstalled. If you prefer this, you can also launch this codespace in the Visual Studio Code desktop client.
+Once loaded, you will be starting at the root of the local repository. You need to do the following to ensure a smooth run-through of the tutorial. First, load the git submodules as it isn't automatically pre-loaded:
+
+```bash
+git submodule update --init --recursive
+```
+
+Second, you need to load some pre-requisite packages:
+
+```bash
+pip3 install -r sw/dnn/requirements.txt
+```
```

Comment:
Why do we need this???

Reply:
My bad I took it out haha. I thought it was required because I couldn't build SW at first. When I re-ran the builds it was working without this LOL.